### PR TITLE
Update docker-compose.yml

### PR DESCRIPTION
**Pull request by tfnribeiro**
- Updated the docker-compose file to use ES8 and to allow developers to run the API without going through apache (logs will be directly onto Docker)
- Added the embedding-api

### CODE REVIEW

**Comment by tfnribeiro** [comment]
Not sure why the tests are failing:

{4A0D44DF-4827-4140-87E5-8691225D469F}

I haven't altered any of the code related to the zeeguu.core?

**Review by mircealungu — APPROVED** [review]
a few comments for you @tfnribeiro

File: docker-compose.yml [hunk 1]
```diff
@@ -30,7 +30,22 @@ services:
...
+      - discovery.type=single-node
+      - xpack.security.enabled=false
+    volumes:
+      - ./data/elasticsearch_db_v8/data:/usr/share/elasticsearch/data
+    networks:
+      - zeeguu_backend
+    restart: unless-stopped
+    mem_limit: 512m # Useful to restrict the ammount of RAM used by ES.
 
   elasticsearch:
```
[the remark this answers is not in the document]
Comment (tfnribeiro) [reply docker-compose.yml]: Agreed!

**Comment by tfnribeiro** [comment]
I have removed the docker-compose-v8 as it is no longer needed, and removed the old ES7 service.

**Pull request merged**